portfolio/src/main/webapp/script.js

```diff
@@ -34,20 +38,62 @@ function addCyclicGreeting() {
 }
 
 /**
- * Fetches and adds a history of comments, and the theoretical maximum and default
- * number of comments, to the page.
+ * Turns on (off) the comment history section if the current setting is
+ * off (on), and reload the comments (either removing or restoring the
+ * comments).
  */
-async function addComments() {
+async function turnOnOffCommentHistorySection() {
```

Comment:
I'd maybe call this "toggleCommentHistorySection()"

Reply:
Gotcha! I'll change it. Thank you